youtube_dl/extractor/soundcloud.py

```diff
@@ -222,6 +228,14 @@ def _real_extract(self, url):
 
         track_id = mobj.group('track_id')
         token = None
+
+        #Get the tracks license, could be improved.
+        track_webpage = self._download_webpage(url, track_id) 
+        track_license = self._html_search_regex(
+            r'</ul>\s*License: (.*)\s*</footer>',
+            track_webpage, 'license', default="all-rights-reserved")
```

Comment:
There is no such licence `all-rights-reserved` exist. Lack of information should be expressed as `None`.